docs/get-started.storybook.mdx

```diff
@@ -0,0 +1,70 @@
+import { Meta } from '@storybook/blocks';
+import WaveLogo from './assets/wave-ds-logo.svg';
+import ByFreenow from './assets/by-freenow.svg';
+
+import { Box } from '../src/components/Box/Box';
+import { Card } from '../src/components/Card/Card';
+import { Headline } from '../src/components/Headline/Headline';
+import { Text } from '../src/components/Text/Text';
+
+<Meta title="Get Started" />
+
+{/* Logo */}
```

Comment:
I'd consider removing this and the blank space below, if this does not break the layout